sdk/communication/azure-communication-network-traversal/README.md

```diff
@@ -0,0 +1,82 @@
+# Azure Communication Network Traversal Package client library for Python
```

Comment:
Can you fix the spacing and line lines in this doc - there's a bunch of places with empty lines. In general, after every heading (`#`, `##`, etc.) there should be a new line, there shouldn't be heading/trailing new lines in code snippets.